src/components/GamesContainer/GameCard/GameCardInfo/GameCardInfoAtributes/GameCardInfoAtributes.jsx

```diff
@@ -0,0 +1,10 @@
+import './GameCardInfoAtributes.css';
```

Comment:
Remember to import styles with CSS modules for not having duplicated classes in the project

Reply:
@fgomensoro I don't understand how to do that?
Not sure if you mean:
import sytle from '...
or if you mean 
rename the file, adding the word module.

Could you explain it please?

Reply:
yes!

like this for example:

import styles from './button.module.css'

const Component = () => (
    <>
        <button className={styles.green}>I am green!</button>
        <button className={styles.blue}>I am blue!</button>
        <button className={styles.red}>I am red!</button>
    </>
)